src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,49 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = "1,2")
+    void splitTest(String input) {
+        String[] split = input.split(",");
+        assertThat(split).isEqualTo(new String[]{"1", "2"});
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = "1")
+    void split_one(String input) {
+        String[] split = input.split(",");
+        assertThat(split).containsExactly("1");
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = "(1,2)")
+    void substringTest(String input) {
+        String substring = input.substring(1).substring(0, input.length()-2);
+        assertThat(substring).isEqualTo("1,2");
+    }
+
+    @DisplayName("특정 위치의 문자를 가져온다.")
+    @ValueSource(strings = "abc")
+    @ParameterizedTest
+    void charAt(String input) {
+        assertThat(input.charAt(0)).isEqualTo('a');
```

Comment:
```suggestion
        assertThat(input.charAt(0)).isEqualTo('a');
        assertThat(input.charAt(1)).isEqualTo('b');
        assertThat(input.charAt(2)).isEqualTo('c');
```
저는 이렇게 다 테스트를 하곤 하는데요 이때 ParameterizedTest 를 활용하면 좋을 것 같아요

Reply:
```
assertAll(
                () -> assertThat(input.charAt(0)).isEqualTo('a'),
                () -> assertThat(input.charAt(1)).isEqualTo('b'),
                () -> assertThat(input.charAt(2)).isEqualTo('c')
        );
```
정욱님 코멘트 참고해서 assertAll 이라는 걸 발견했습니다ㅎㅎㅎ

Reply:
@wisdom08 
assertAll 을 저도 지혜님 덕에 알게되어 조금 찾아보았는데요
assertThat(input.charAt(0)).isEqualTo('a') -> 실패할 경우
assertThat(input.charAt(1)).isEqualTo('b') -> 실행 안함

assertAll 로 그룹화를 했을 땐 모두 다 실행을 한다고 합니다!!!
적절한 경우가 생기면 되게 좋을 것 같은 메소드에요 감사합니다👍